backend/main.go

```diff
@@ -28,11 +42,24 @@ func main() {
 		}
 	}(server)
 
-	server.Port = 8080
-	server.Host = "0.0.0.0"
+	server.Port = cfg.Port
+	server.Host = cfg.Host
 
 	if err := server.Serve(); err != nil {
 		log.Panic().Err(err).Msg("server error")
 	}
 
 }
+
+func initLog(cfg *Config) {
+	var err error
+
+	host, err = os.Hostname()
+	if err != nil {
+		log.Panic().Err(err).Msg("unable to get hostname")
+	}
+
+	zerolog.SetGlobalLevel(zerolog.DebugLevel)
```

Comment:
If we are in debug mode, lets set the level to `Debug`, otherwise set it to `Info`.

Reply:
im not sure what you mean by set it to info? is this a flag i should have in my .env file?